addons/medical_core/scripts/Game/@ACE_Medical_Core/Settings/ACE_Medical_Core_Settings.c

```diff
@@ -4,7 +4,10 @@ class ACE_Medical_Core_Settings : ACE_ModSettings
 {
 	[Attribute(defvalue: "0.3", desc: "Default resilience regeneration scale. The default regeneration rate (5 HP/s) will be multiplied by this factor.", params: "0 inf")]
 	float m_fDefaultResilienceRegenScale;
-	
+
+	[Attribute(defvalue: "1.0", desc: "Resilience damage scale. All incoming damage (except regeneration) to the resilience hitzone will be multiplied by this factor. Lower values make characters more resilient.", params: "0 inf")]
```

Comment:
```suggestion
	[Attribute(defvalue: "1.0", desc: "Resilience damage scale. All incoming damage to the resilience hitzone will be multiplied by this factor. Lower values make characters less likely to fall unconscious.", params: "0 inf")]
```
This is redundant, while internally a damage type, the users don't really need to know that.